starport/services/chain/app.go

```diff
@@ -3,10 +3,14 @@ package chain
 import (
 	"path/filepath"
 	"strings"
+
+	"github.com/tendermint/starport/starport/pkg/cosmosver"
 )
 
 // App keeps info about scaffold.
```

Comment:
It's not only related to scaffold anymore, isn't it?

Reply:
Yep.
```suggestion
// App keeps info about chain.
```